webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -21,18 +21,22 @@ export const getNotifications = url => dispatch => {
       document.visibilityState === 'visible' ||
       document.visibilityState === 'prerender'
     ) {
-      API.get(url).then(
-        response => {
+      API.get(url)
+        .success(response => {
```

Comment:
The jqXHR.success(), jqXHR.error(), and jqXHR.complete() callbacks are removed as of jQuery 3.0 (http://api.jquery.com/jQuery.ajax/), so we'd better not use them.